app/scripts/components/exploration/components/datasets/colormap-options.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Icon } from "@trussworks/react-uswds";
+import { CollecticonDrop } from '@devseed-ui/collecticons';
+import { sequentialColorMaps, divergingColorMaps, restColorMaps } from './colorMaps';
+
+import './colormap-options.scss';
+
+const CURATED_SEQUENTIAL_COLORMAPS = [
+  'viridis', 'plasma', 'inferno', 'magma', 'cividis',
+  'purples', 'blues', 'reds', 'greens', 'oranges',
+  'ylgnbu', 'ylgn', 'gnbu'
+];
+
+const CURATED_DIVERGING_COLORMAPS = [
+  'rdbu', 'rdylbu', 'bwr', 'coolwarm'
+];
+
+export const classifyColormap = (colormapName: string): 'sequential' | 'diverging' | 'rest' | 'unknown' => {
+  const baseName = normalizeColorMap(colormapName);
+
+  if (sequentialColorMaps[baseName]) {
+    return 'sequential';
+  } else if (divergingColorMaps[baseName]) {
+    return 'diverging';
+  } else if (restColorMaps[baseName]) {
+    return 'rest';
+  }
+  return 'unknown';
+};
+
+interface ColormapOptionsProps {
+  colorMap: string;
+  setColorMap: (colorMap: string) => void;
+}
+
+export const getColormapColors = (colormapName: string, isReversed: boolean): string[] => {
+  const baseName = normalizeColorMap(colormapName);
+  const colormapData =
+    sequentialColorMaps[baseName] ||
+    divergingColorMaps[baseName] ||
+    restColorMaps[baseName] ||
+    sequentialColorMaps.viridis;
+
+  const colorKeys = Object.keys(colormapData);
+  const colors = colorKeys.map((key) => {
+    const [r, g, b, a] = colormapData[key];
+    return `rgba(${r}, ${g}, ${b}, ${a})`;
+  });
+
+  return isReversed ? colors.reduceRight((acc, color) => [...acc, color], []) : colors;
+};
+
+export function ColormapOptions({ colorMap, setColorMap}: ColormapOptionsProps) {
```

Comment:
Curious, with Jotai state management, cant we access the same state from the same atom? so instead of passing `colorMap` state and setter method, can we just pass in the `datasetAtom` and then do `const [colorMap, setColorMap] = useTimelineDatasetColormap(datasetAtom);` at this level to avoid prop drilling since `datasetAtom` was already being previously being passed into `DataLayerCard`?